packages/components/src/FormField/FormField.tsx

```diff
@@ -232,6 +232,7 @@ export function FormField({
   const wrapperClassNames = classnames(
     styles.wrapper,
     inline && styles.inline,
+    inline && type === "textarea" && styles.inlineTextArea,
```

Comment:
If we drop text area support this should no longer be needed.

Reply:
Removed